rust/src/measurements/gumbel_max/mod.rs

```diff
@@ -142,11 +148,15 @@ where
 
     (iter.enumerate())
         .map(|(i, v)| {
-            let mut shift = v.into_rational()?;
+            let mut shift = FBig::try_from(v).unwrap_or(FBig::ZERO);
```

Comment:
I don't know what kind of failures are expected, so I'm not sure that zero always makes sense as a fallback. Unless it would be obvious to most readers, add a comment?

Reply:
We make the assumption across the library that the input data is a member of the input domain. 

If the input data is in the input domain, then this won't fail, because it only fails if `v` is nan. The thinking here is that, if the input score is nan, then the algorithm fails more gracefully. I can do a bit better by just dropping nans/failures to convert from the iterator.